lib/ui/shared/dumb_widgets/zuri_appbar.dart

```diff
@@ -25,6 +25,9 @@ class ZuriAppBar extends StatelessWidget implements PreferredSizeWidget {
   bool isSearchBar;
   final String? subtitle;
   final List<Widget>? actions;
+  final FocusNode? focusNode;
```

Comment:
@jeremiahseun Please review if the changes to this file are fine by you

Reply:
It's cool